.gitignore

```diff
@@ -81,6 +81,7 @@ e2e/fixtures/cache/
 
 test-api.ts
 packages/package-tests/**/package-lock.json
+scratchpad/
```

Comment:
just a folder for manual testing, I think it is nice to have one. Not sure about naming